fiftyone/core/metadata.py

```diff
@@ -382,28 +389,79 @@ def compute_metadata(
             default value ``fiftyone.config.show_progress_bars`` (None), or a
             progress callback function to invoke instead
     """
-    num_workers = fou.recommend_thread_pool_workers(num_workers)
+    logger.info("Computing metadata...")
```

Comment:
The previous implementation intentionally only logged this message if there were actually >0 samples that needed metadata computed. I'd like to maintain that behavior. The underlying convention I'd like to follow is to only generate logs in core methods when things actually happen, not when they don't.

Reply:
Okay. I only added it because on large datasets it would previously hang with no user feedback of what was happening.

Reply:
Actually, I don't see anywhere else I can log it since all the rest is checked internal to the mapper so I just changed it to debug.